### ISSUE TITLE
Icon?

### ISSUE THREAD

**Issue by MuntashirAkon** [MuntashirAkon/unapkm-android#1]
It would be great if somebody could suggest me a good icon (I'm very bad at things like this).

**Comment by xballiet** [comment]
Something like this ?
![UnAPKM](https://user-images.githubusercontent.com/5589530/99784476-254fd680-2b1c-11eb-9a64-740daf5fe898.png)

**Comment by MuntashirAkon** [comment]
Seems to be a nice idea: APKM -> APKS. But the design can be improved.

**Comment by xballiet** [comment]
Give me your ideas to improve, I will see what I can do.  :D

**Comment by MuntashirAkon** [comment]
> Give me your ideas to improve, I will see what I can do. :D

I'm really not good at these things. To me the problem with the above logo was that the colours didn't and the placement of the split icon didn't look very well. Since the app is some sort of converter, maybe you can get idea from a converter or a migration app.

**Comment by xballiet** [comment]
Could this be better for you?
![UnAPKM](https://user-images.githubusercontent.com/5589530/100123957-56613b80-2e7b-11eb-9a6e-5b4e9e7458c0.png)

**Comment by MuntashirAkon** [comment]
> Could this be better for you?

Could be. But I'm not so sure how APK Mirror will think of it though since you're using their design which is under a non-free license.

**Comment by xballiet** [comment]
OK, but then I don't what to use instead... an idea?

**Comment by xballiet** [comment]
Or perhaps you could ask them directly if they are OK with this icon?

**Comment by MuntashirAkon** [comment]
> OK, but then I don't what to use instead... an idea?

I really have no idea, sorry! But here's what UnApkm does (it might help in forming new ideas): it _decrypts_ an encrypted APKM file and saves it as APKS.

**Comment by xballiet** [comment]
What do you think about this?
![UnAPKM](https://user-images.githubusercontent.com/5589530/100333266-ab639580-2fd2-11eb-85f4-cdf01d556848.png)

**Comment by MuntashirAkon** [comment]
> What do you think about this?

Yes, close, I think. Consider changing the colour palette as the white doesn't look standout in orange/brown.

**Comment by xballiet** [comment]
Which colour palette do you prefer?
And do you want me to replace white (and grey) for the text only or for the spinning arrow around the text also?

**Comment by xballiet** [comment]
Could this, with blueish colours, be more readable?

![UnAPKM](https://user-images.githubusercontent.com/5589530/100861147-b6666c00-3491-11eb-89be-768890fe6bdb.png)

**Comment by MuntashirAkon** [comment]
Ok, I'm finally a bit free to take an in-depth look at your efforts in making a nice logo for UnAPKM. Here are my opinions:

1. Many people still likes rounded square icons. So, please make the icon rounded square keeping the icon as is.
2. Shadow makes the icon more complicated. Please keep it simple and remove shadows.
3. Light colours look better on dark colours and vice versa. The colours used here are not very distinguishable in terms of brightness. If you're looking for good colour combinations, you can take a look at here: https://inspirationfeed.com/good-color-combinations

I'm adding my preferences here (from the above link):

![](https://inspirationfeed.com/wp-content/uploads/2015/08/colors15-620x265.png)

![](https://inspirationfeed.com/wp-content/uploads/2015/08/colors26-620x265.png)

![](https://inspirationfeed.com/wp-content/uploads/2015/08/colors-7-620x265.png)

![](https://inspirationfeed.com/wp-content/uploads/2015/08/colors24-620x265.png)

Thanks for your efforts 👍

**Comment by xballiet** [comment]
OK, I have totally reviewed the icon, how do feel with this one?

![UnAPKM](https://user-images.githubusercontent.com/5589530/100868946-9b4d2980-349c-11eb-83fd-8e9c85f0a6f0.png)

**Comment by MuntashirAkon** [comment]
> OK, I have totally reviewed the icon, how do feel with this one?

Looks cool! But the colour brightness is still an issue. Better replace pink-reddish with more dark colours/sheds and I think it's better to use a cross than a strikethrough.

**Comment by xballiet** [comment]
I have used the last of the four palettes you have "preferenced", do you prefer another one?
If we keep this form, just tell me which colour of the chosen palette to put in each "contiguous part" of the icon (the outer border, the background, the arrows and the texts)

As for the APKM's "M", I am not quite OK with you: for me, putting a cross on it will make it less readable, but I can also put this M in another colour to emphatize it?

**Comment by MuntashirAkon** [comment]
> I have used the last of the four palettes you have "preferenced", do you prefer another one?

Those were just suggestions tbh. I'm not expert in choosing colours.


> If we keep this form, just tell me which colour of the chosen palette to put in each "contiguous part" of the icon (the outer border, the background, the arrows and the texts)

Adding darker sheds to the pink-reddish colour is enough, I think.

> As for the APKM's "M", I am not quite OK with you: for me, putting a cross on it will make it less readable, but I can also put this M in another colour to emphatize it?

Yeah, I can realise that now.

**Comment by xballiet** [comment]
OK, here are four propositions, tell me if one of them has your preference or if I can adjust something else:

![UnAPKM1](https://user-images.githubusercontent.com/5589530/100890034-18d26300-34b8-11eb-951f-56c7ca925c70.png)

![UnAPKM2](https://user-images.githubusercontent.com/5589530/100890047-1bcd5380-34b8-11eb-8d82-5b3e80dd9768.png)

![UnAPKM3](https://user-images.githubusercontent.com/5589530/100890054-1e2fad80-34b8-11eb-88e4-7c48e37cc9e1.png)

![UnAPKM4](https://user-images.githubusercontent.com/5589530/100890064-212a9e00-34b8-11eb-8f3f-a3999e230627.png)

**Comment by MuntashirAkon** [comment]
First one looks good.

---
_Note that the chosen icon will be used both in here and in my other project, App Manager, which has an activity that offers the same feature._

**Comment by xballiet** [comment]
I'm glad we have been able to make this together, enjoy the new icon!  ;)

**Comment by Dhyfer1** [comment]
> I'm glad we have been able to make this together, enjoy the new icon!  ;)

your icon is very ugly and boring 🙄
